scripts/testing/runtest

```diff
@@ -140,7 +140,11 @@ def _options():
     )
 
     config = cp.ConfigParser(defaults)
-    config.read(args[0])
+    try:
+        with open(args[0]) as cf:
+            config.read_file(cf)
+    except OSError:
+        parser.error('there was an error opening the configuration file')
```

Comment:
Be more informative:

```python
except OSError as e:
  parser.error(f'blablabla: {e.strerror}')
```

Reply:
The full error message now reads:

`runtest: error: while reading configuration from '<path>': <error string>`